src/main/scala/mesosphere/mesos/ResourceMatcher.scala

```diff
@@ -133,7 +134,7 @@ object ResourceMatcher extends StrictLogging {
     * the reservation.
     */
   def matchResources(offer: Offer, runSpec: RunSpec, knownInstances: => Seq[Instance],
-    selector: ResourceSelector, conf: MatcherConf, schedulerPlugins: Seq[SchedulerPlugin] = Seq.empty)(implicit clock: Clock): ResourceMatchResponse = {
+    selector: ResourceSelector, conf: MatcherConf, schedulerPlugins: Seq[SchedulerPlugin] = Seq.empty, homeFaultDomain: Option[FaultDomain])(implicit clock: Clock): ResourceMatchResponse = {
```

Comment:
Ok, this is a little confusing. I thought this diff is about testing the workload bursting. However, it seems we introduce the feature here. Could you shed a little light on this? 🙂

Reply:
I agree. I moved all feature-related code to another PR: https://github.com/mesosphere/marathon/pull/5673